docker.md

```diff
@@ -0,0 +1,57 @@
+# Fooocus on Docker
+
+It's based on NVIDIA CUDA 12.3. See [Dockerfile](Dockerfile) for details.
+
+PyTorch version is 2.1. See [requirements_docker.txt](requirements_docker.txt) for details.
+
+## Quick start
+
+**This is just an easy way for testing. Please also see [Notes](#notes) for using.**
+
+Clone git, Then run the docker container `docker compose up --build`
+It takes a time to build the container image.
+When you see a messeage  `Use the app with http://localhost:7865/` in the console message, you can access the URL on your browser.
+
+Your models, outputs are stored into 'fooocus-data' volume which may be stored in '/var/lib/docker/volumes'.
+
+## Use pre-built image
+
+The pre-built image is provided at `whitehara/fooocus`
```

Comment:
Same comment as above.

Reply:
we could even automate the pipeline for each main branch update (or directly optimise thr process and use Gitlab releases for that...) and tag it with the corresponding version

Reply:
> Same comment as above.

If someone creates an official dockerhub account, I can modify the document for that. Then, I will stop providing my images in my dockerhub account.

Reply:
> we could even automate the pipeline for each main branch update (or directly optimise thr process and use Gitlab releases for that...) and tag it with the corresponding version

I'm doing it in my repo. Please refer my [Github Actions](https://github.com/whitehara/Fooocus/blob/docker-build/.github/workflows/docker-publish.yml).

Reply:
Sorry for the drive-by comment. I hadn't noticed you were already publishing the images to dockerhub. That makes sense. It wasn't my intention to suggest adding publishing images for this repo in this PR. Although that would be great! 

I was just getting impatient 🤭. Thanks for your work!

Reply:
@lllyasviel would you be willing to create a Dockerhub repository for Fooocus or shall i set it up + add the according actions to Github pushes (main) and/or releases?